server/src/api/v1/users/auth-user.ts

```diff
@@ -39,6 +41,8 @@ export async function loginUser(req: Request, res: Response): Promise<Response>
         const passwordMatches = await bcrypt.compare(incomingData.password, user.password)
             .catch(() => false);
         if (passwordMatches) {
+            const userHasAdminRole = user.assigned_roles.findIndex(el => el.id === user.organization.admin_role_id)+1;
+            Vars.currentUserIsAdmin = !!userHasAdminRole;
```

Comment:
Leserlicher wäre doch:

```suggestion
            // check if any of the user's roles are the current user's organisation's admin role
            Vars.currentUserIsAdmin = user.assigned_roles.find(role => role.id === user.organization.admin_role_id) !== undefined;
```

(ungetestet)